components/common/ErrorModal.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button, Text, Overlay } from "react-native-elements";
+import { StyleSheet } from "react-native";
+import { FC, useState } from "react";
+
+type Props = { onRemove: any };
+
+const ErrorModal: FC<Props> = ({ onRemove }) => {
+  const removeErrorModalHandler = () => {
+    onRemove();
+    setIsVisible(false);
+  };
+  const [isVisible, setIsVisible] = useState<boolean>(true);
+
+  return (
+    <Overlay onBackdropPress={removeErrorModalHandler} isVisible={isVisible}>
+      <Text style={styles.title}>Sorry!</Text>
+      <Text style={styles.text}>Something Went wrong!</Text>
```

Comment:
message should be a little more detail...?

we can throw error message like this.

and is it possible we show error from this...?

```
      if (!response.ok) {
        const data = await response.json();
        console.log("data", data.error.message);
        throw new Error(data.error.message);
      }
```

Reply:
Yeah I think we can change the message. But can we get data when we are in the !res.OK block?